default.nix

```diff
@@ -49,21 +49,29 @@ let haskellPackages =
                         mkPackage super "streamly-examples"
                             ./. flags inShell;
 
-                    streamly =
-                      nixpkgs.haskell.lib.overrideCabal
-                        (super.callHackageDirect
-                          { pkg = "streamly";
-                            ver = "0.8.0";
-                            sha256 = "0vy2lkljizlhpbpbybmg9jcmj2g4s1aaqd2dzy5c0y0n4rgwxask";
-                          } {})
-                        (old:
-                          { librarySystemDepends =
-                              if builtins.currentSystem == "x86_64-darwin"
-                              then [nixpkgs.darwin.apple_sdk.frameworks.Cocoa]
-                              else [];
-                            enableLibraryProfiling = false;
-                            doHaddock = false;
-                          });
+                    # TODO: Uncomment the following after the streamly release
+
+                    # streamly =
+                    #   nixpkgs.haskell.lib.overrideCabal
+                    #     (super.callHackageDirect
+                    #       { pkg = "streamly";
+                    #         ver = "0.8.0";
+                    #         sha256 = "0vy2lkljizlhpbpbybmg9jcmj2g4s1aaqd2dzy5c0y0n4rgwxask";
+                    #       } {})
+                    #     (old:
+                    #       { librarySystemDepends =
+                    #           if builtins.currentSystem == "x86_64-darwin"
+                    #           then [nixpkgs.darwin.apple_sdk.frameworks.Cocoa]
+                    #           else [];
+                    #         enableLibraryProfiling = false;
+                    #         doHaddock = false;
+
+                    # unicode-data =
+                    #   super.callHackageDirect
+                    #     { pkg = "unicode-data";
+                    #       ver = "0.2.0";
+                    #       sha256 = "073wbhdxj1sh5160blaihbzkkhabs8s71pqhag16lvmgbb7a3hla";
+                    #     } {};
```

Comment:
Why do we need to even comment it? Won't cabal override it from the cabal.project file?

Reply:
I've been having some nix errors that keeps saying `unable to find unicode-data` although I add this. It does even check the `sha256`. For the time being, I don't want to debug that.
Some caching issue probably.

Reply:
I had this working on the split branch (or may be locally let me check).

Reply:
Add this in additionalDeps:

```
              executableHaskellDepends = with hspkgs; [
                unicode-data
              ];
```

Reply:
I still have the same issue.

Reply:
Please ignore this. This will be reverted anyway.

Reply:
Maybe you have the sha256 of some other package? In that case it successfully matches the package, does not give any error and uses the wrong package instead.